src/monitor/pgautofailover.sql

```diff
@@ -111,6 +112,7 @@ CREATE TABLE pgautofailover.node
     replicationquorum	 bool not null default true,
 
     UNIQUE (nodename, nodeport),
+    EXCLUDE USING gist(groupid with =, sysidentifier with <>),
```

Comment:
Can we add a comment explaining the goal of this constraint? You need quite some background to understand this without context. Maybe giving the constraint a user friendly name is also nice, since it is displayed in the error message.